tests/test_formparser.py

```diff
@@ -138,6 +140,16 @@ def test_parse_form_data_get_without_content(self):
         strict_eq(len(form), 0)
         strict_eq(len(files), 0)
 
+    @pytest.mark.parametrize('size,expected', [
+        (-1, BytesIO),
+        (0, BytesIO),
+        (1024 * 500, BytesIO),
+        (1024 * 501, file),
+        (maxint, file),
+    ])
+    def test_default_stream_factory(self, size, expected):
+        assert isinstance(default_stream_factory(size, '', ''), expected)
```

Comment:
I removed this test in master because `BytesIO` is not necessarily a class. Thanks though!